book/06-github/sections/2-contributing.asc

```diff
@@ -270,11 +270,11 @@ To https://github.com/tonychacon/blink
    ef4725c..3c8d735  slower-blink -> slow-blink
 ----
 
-<1> Das Original-Repository als Remote mit der Bezeichnung „upstream“ hinzufügen
-<2> Die neueste Arbeit von diesem Remote abrufen (engl. fetch)
+<1> Das Original-Repository als Remote mit der Bezeichnung „upstream“ hinzufügen.
+<2> Die neueste Arbeit von diesem Remote abrufen (engl. fetch).
 <3> Den Haupt-Branch dieses Repositorys mit dem Themen-Branch mergen.
-<4> Den aufgetretenen Konflikt beheben
-<5> Zum gleichen Themen-Branch zurück pushen
+<4> Den aufgetretenen Konflikt beheben.
+<5> Zum gleichen Themen-Branch zurück pushen.
```

Comment:
### Betr: alle Listen- und Tabellenzeilen
Ich bin unentschieden wie einzelne Zeilen in Listen zu behandeln sind (auch der englische Text ist da nicht eindeutig).

In Aufzählungen und Tabellen sind die Texte der Zeilen oft nicht als kompletter Satz formuliert – dann gehört dort kein abschließender Punkt hin. Andererseits haben die englischen Autoren sowohl komplette Sätze als auch Rumpfsätze wechselseitig mit und ohne Punkt verwendet.

Ich tendiere eher dazu in Aufzählungen/Tabellen **keine** abschließende Punkte zu verwenden.

Reply:
Ob mit oder ohne Punkt ist für mich eher zweitrangig, wichtiger währe mir das sich für eine Version entschieden wird und diese verwendet wird. Gerade wen in einer Liste mal mit und mal ohne Punkt geschrieben wird sieht das etwas komisch aus.

Reply:
Genau,
wobei ich keine Bauchschmerzen hätte, wenn das bei unterschiedlichen Listen jeweils anders gehandhabt wird. Nur innerhalb der Liste sollte es einheitlich sein.
Mich stört der abschließende Punkt eigentlich nur dann, wenn die Listenzeile nur aus wenigen Schlagworten besteht.